.github/dependabot.yml

```
@@ -12,7 +12,7 @@ updates:
     labels:
       - "type: dependencies"
     reviewers:
-      - "honeycombio/telemetry-team"
+      - "honeycombio/Pipeline-team"
```

```suggestion
      - "honeycombio/pipeline-team"
```

IDK if caps matters but just in case